arrow/src/main/java/org/apache/iceberg/arrow/vectorized/VectorizedReaderBuilder.java

```diff
@@ -97,7 +98,7 @@ public VectorizedReader<?> message(
       } else if (reader != null) {
         reorderedFields.add(reader);
       } else {
-        reorderedFields.add(VectorizedArrowReader.nulls());
```

Comment:
based on my other comments, this should most likely call `VectorizedArrowReader.nulls(field)`